### DISCUSSION TITLE
Event not show in event tab firebase console

### DISCUSSION THREAD

**Discussion by hwq0521** [invertase/react-native-firebase#4777] [Q&A]
Hi, I'm unable to found the triggered events in firebase console, while can found those events in GA4 -> Tech -> Tech Details.
I have **two firebase project** which differ in **production** and **UAT**. I have tested in these two project with debug view, I'm sure that events had shown in both project's Debug View, but the event tab in **firebase project production** doesn't has those events, but the **firebase project UAT** has those events. May I know what's the problem?

**Comment by WadhahEssam** [comment]
Try this if you are using the latest version.

- set `analytics_auto_collection_enabled` in the firebase.json to true
```
{
  "react-native": {
    "admob_android_app_id": "XXX",
    "admob_ios_app_id": "XXX",
    "analytics_auto_collection_enabled": true
  }
```

- OR call setAnalyticsCollectionEnabled();
```
import { firebase } from '@react-native-firebase/analytics';
// ...
await firebase.analytics().setAnalyticsCollectionEnabled(true);
```

**Reply by hwq0521** [reply]
Hi, thanks for your reply. I'm using this version, "@react-native-firebase/analytics": "^7.1.0", previously is working, and I have tried in another firebase project with same code, and the another firebase project is working. Hence, no idea why it's not working in my current firebase project.
